tests/test_livysession.py

```diff
@@ -56,7 +57,9 @@ def _create_session(self, kind=Constants.session_kind_spark, session_id="-1", sq
         if http_client is None:
             http_client = MagicMock()
 
-        return LivySession(http_client, session_id, sql_created, {"kind": kind})
+        ipython_display = MagicMock()
+
+        return LivySession(ipython_display, http_client, session_id, sql_created, {"kind": kind})
```

Comment:
As Ricky pointed out, please verify that `ipython_display` methods are called correctly throughout this class.